modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -26,14 +26,10 @@ enum class StereoOutputFormat {
     DISPARITY_FIXED16_12_4, ///< 16 bit signed: first bit for sign,
                             ///< 11 bits for integer part,
                             ///< 4 bits for fractional part.
-    DEPTH_F16, ///< Floating point 16 bit value, CV_16FC1
-    DEPTH_F32, ///< Floating point 32 bit value, CV_32FC1
-    DISPARITY_Q16_10_5, ///< 16 bit signed: first bit for sign,
-                        ///< 10 bits for integer part,
-                        ///< 5 bits for fractional part.
-    DISPARITY_Q16_11_4  ///< 16 bit signed: first bit for sign,
-                        ///< 11 bits for integer part,
-                        ///< 4 bits for fractional part.
+    DEPTH_16F = DEPTH_FLOAT16, ///< Same as DEPTH_FLOAT16
+    DEPTH_32F = DEPTH_FLOAT32, ///< Same as DEPTH_FLOAT32
+    DISPARITY_16Q_10_5 = DISPARITY_FIXED16_11_5, ///< Same as DISPARITY_FIXED16_11_5
+    DISPARITY_16Q_11_4 = DISPARITY_FIXED16_12_4 ///< Same as DISPARITY_FIXED16_12_4
```

Comment:
> Same as

Perhaps, it makes sense to describe which variant is `deprecated`.

Reply:
+1

Reply:
Done